error while building the bumblebee package

This is the output of make after the abort:

gcc -Wall -Waggregate-return 	-Wmissing-declarations -Wmissing-prototypes -Wredundant-decls 	-Wshadow -Wstrict-prototypes -Winline -Wformat=2 -pipe   -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -Wextra -funsigned-char -DGITVERSION='"3.2.1-2013-04-26-Format:%h$"' -O2 -fPIC   -o bin/optirun src/module.o src/bbconfig.o src/bblogger.o src/bbrun.o src/bbsocket.o src/driver.o src/optirun.o src/bbsocketclient.o -lglib-2.0 -lrt 
/usr/bin/ld: src/bblogger.o:(.bss+0x220): multiple definition of `bb_pm_method_string'; src/bbconfig.o:(.data.rel.local+0x0): first defined here
/usr/bin/ld: src/bbsocket.o:(.bss+0x0): multiple definition of `bb_pm_method_string'; src/bbconfig.o:(.data.rel.local+0x0): first defined here
/usr/bin/ld: src/driver.o:(.bss+0x0): multiple definition of `bb_pm_method_string'; src/bbconfig.o:(.data.rel.local+0x0): first defined here
/usr/bin/ld: src/optirun.o:(.bss+0x0): multiple definition of `bb_pm_method_string'; src/bbconfig.o:(.data.rel.local+0x0): first defined here
/usr/bin/ld: src/bbsocketclient.o:(.bss+0x0): multiple definition of `bb_pm_method_string'; src/bbconfig.o:(.data.rel.local+0x0): first defined here
collect2: Error: ld returned 1 as end state
make: *** [Makefile:604: bin/optirun] error 1

--- a/bumblebee/bumblebee.SlackBuild
+++ b/bumblebee/bumblebee.SlackBuild
@@ -25,7 +25,7 @@
 
 PRGNAM=bumblebee
 VERSION=${VERSION:-$(echo ${PRGNAM}-*.tar.?z* | rev | cut -f 3- -d . | cut -f 1 -d - | rev)}
-BUILD=${BUILD:-1}
+BUILD=${BUILD:-2}
 TAG=${TAG:-_bbsb}
 
 if [ -z "$ARCH" ]; then
@@ -82,6 +82,9 @@ fi
 # Default to primus over VirtualGL
 patch -p1 < $CWD/patches/optirun-default-to-primus-then-vgl.patch
 patch -p1 < $CWD/patches/updated-lib32-to-lib64-for-primus-path.patch
+
+# Patch to compile in GCC 10
+patch -p1 < $CWD/patches/bumblebee-ggc10.patch
 
 autoreconf -fi
 

--- a/bumblebee/patches/bumblebee-ggc10.patch
+++ b/bumblebee/patches/bumblebee-ggc10.patch
@@ -1,0 +1,67 @@
+Author: Andreas Beckmann <anbe@debian.org>
+Description: fix FTBFS with gcc-10
+ gcc-10 defaults to -fno-common
+ see https://gcc.gnu.org/gcc-10/porting_to.html
+Bug-Debian: https://bugs.debian.org/957058
+
+--- a/src/bbconfig.h
++++ b/src/bbconfig.h
+@@ -103,7 +103,7 @@ enum bb_pm_method {
+     PM_VGASWITCHEROO,
+     PM_METHODS_COUNT /* not a method but a marker for the end */
+ };
+-const char *bb_pm_method_string[PM_METHODS_COUNT];
++extern const char *bb_pm_method_string[PM_METHODS_COUNT];
+
+ /* String buffer size */
+ #define BUFFER_SIZE 1024
+--- a/src/switch/switching.h
++++ b/src/switch/switching.h
+@@ -60,10 +60,10 @@ void switcheroo_off(void);
+
+ /* number of switchers as defined in switching.c */
+ #define SWITCHERS_COUNT 2
+-struct switching_method switching_methods[SWITCHERS_COUNT];
++extern struct switching_method switching_methods[SWITCHERS_COUNT];
+
+ /* A switching method that can be used or NULL if none */
+-struct switching_method *switcher;
++extern struct switching_method *switcher;
+
+ struct switching_method *switcher_detect(const char *name, struct switch_info);
+ enum switch_state switch_status(void);
+--- a/src/bbsecondary.h
++++ b/src/bbsecondary.h
+@@ -21,7 +21,7 @@
+ #pragma once
+
+ /* PCI Bus ID of the discrete video card */
+-struct pci_bus_id *pci_bus_id_discrete;
++extern struct pci_bus_id *pci_bus_id_discrete;
+
+ /// Start the X server by fork-exec, turn card on if needed.
+ bool start_secondary(bool);
+--- a/src/switch/switching.c
++++ b/src/switch/switching.c
+@@ -31,6 +31,8 @@ struct switching_method switching_method
+           switcheroo_on, switcheroo_off}
+ };
+
++struct switching_method *switcher = NULL;
++
+ /**
+  * Enumerates through available switching methods and try a method
+  *
+--- a/src/bbsecondary.c
++++ b/src/bbsecondary.c
+@@ -37,6 +37,9 @@
+ #include "pci.h"
+ #include "module.h"
+ 
++/* PCI Bus ID of the discrete video card */
++struct pci_bus_id *pci_bus_id_discrete;
++
+ /**
+  * Substitutes DRIVER in the passed path
+  * @param x_conf_file A path to be processed
+

